.github/CODEOWNERS

```diff
@@ -284,7 +284,8 @@ packages/core/usage-data/core-usage-data-base-server-internal @elastic/kibana-co
 packages/core/usage-data/core-usage-data-server @elastic/kibana-core
 packages/core/usage-data/core-usage-data-server-internal @elastic/kibana-core
 packages/core/usage-data/core-usage-data-server-mocks @elastic/kibana-core
-packages/core/versioning/core-version-http-server @elastic/kibana-core
+packages/core/versioning/core-versioned-http-server @elastic/kibana-core
+packages/core/versioning/core-versioned-http-server-internal @elastic/kibana-core
```

Comment:
we've been *trying* to stick to one public type package per server/browser domain, so I would move the types from `core-versioned-http-server` to `core-http-server` if possible.